packages/spec/renderer.js

```diff
@@ -16,19 +16,35 @@ describe('renderer', function () {
   });
 
   it('should create a renderer function', function () {
-    var render = createRenderer(goodConfig);
+    var render = createRenderer({ webpackConfig: goodConfig });
     assert.equal(typeof render, 'function');
   });
 
   describe('function', function () {
     it('should return a promise', function () {
-      var render = createRenderer(goodConfig);
+      var render = createRenderer({ webpackConfig: goodConfig });
       var promise = render('/');
       assert(promise instanceof Promise);
     });
 
+    it('should call middleware', function () {
+      var render = createRenderer({
+        webpackConfig: goodConfig,
+        hopsConfig: {
+          bootstrapServer: function (app) {
+            assert(true);
+            app.use(function (req, res, next) {
+              assert(true);
+              next();
```

Comment:
The test is green even if `bootstrapServer` is never called, because the test is synchronously.
How about adding a `done()` callback here, so the test will timeout when the middleware is never called?